core/controllers/base.py

```diff
@@ -291,6 +312,13 @@ def render_json(self, values):
     def render_template(
             self, filename, iframe_restriction='DENY',
             redirect_url_on_logout=None):
+        """Renders a template file
+
+        Args:
+            filename: str. Name of the template file
+            iframe_restriction: str. Value for iframe restriction.
```

Comment:
Description doesn't convey anything; please explain allowed values and their significance.